src/generated/type/src/model.rs

```diff
@@ -458,17 +482,27 @@ pub struct Decimal {
     ///
     /// The ENBF grammar is:
     ///
-    ///     DecimalString =
-    ///       [Sign] Significand [Exponent];
+    /// ```norust
+    /// DecimalString =
+    ///   [Sign] Significand [Exponent];
+    /// ```
```

Comment:
I think my hack is working correctly in this case, creating multiple blockquotes:

https://spec.commonmark.org/0.31.2/#example-242

However, this may not have been the intent upstream. We may want to fix the upstream code to use explicit `> ` markers.